README.md

```diff
@@ -28,6 +28,7 @@ gem install fluent-plugin-filter-kv-parser
   filtered_keys akey,bkey,ckey
   filtered_keys_regex /^sub_[a-zA-Z_0-9]+/
   filter_out_lines_without_keys false
+  add_prefix keys_prefix
```

Comment:
```suggestion
  keys_prefix test
```

Reply:
Sorry, I misunderstood your suggestion. I'll change it.